modules/apps/27-interchain-accounts/host/client/cli/tx.go

```diff
@@ -85,19 +97,20 @@ which submits pre-built packet data containing messages to be executed on the ho
 		},
 	}
 
-	cmd.Flags().String(memoFlag, "", "an optional memo to be included in the interchain account packet data")
+	cmd.Flags().String(memoFlag, "", "an optional memo to be included in the interchain accounts packet data")
```

Comment:
for consistency with below

```suggestion
	cmd.Flags().String(memoFlag, "", "optional memo to be included in the interchain accounts packet data")
```